src/foam/u2/stack/StackView.js

```diff
@@ -47,7 +47,12 @@ foam.CLASS({
         this.add(this.Stack.BACK, this.Stack.FORWARD);
 
       this.add(this.slot(function(s) {
-        return foam.u2.ViewSpec.createView(s, null, this, this.__subSubContext__);
+        var view = s[0];
+        var parent = s[1];
+        var X = parent ? this.__subSubContext__.createSubContext(parent) : this.__subSubContext__;
```

Comment:
If I push something onto the stack, I want it to be able to pick up things that I have exported, but I also want it to be in a sub-context of its parent node like always.

This was the motivation for doing Context Multiple Inheritence.

For instance DAOControllerView exports 'data.dao as dao', and DAOUpdateControllerView imports 'dao'

Without this, DAOUpdateController would only pick up the context of the stack view it's being rendered in and be unable to find it's 'dao' import.